### PR TITLE
CMSIS-NN: Multiple batches depthwise

### PR DESCRIPTION
**Pull request by mansnils**
* Adds multiple batch support to depthwise_conv_s8_generic and update
      wrapper to reflect that.
    * Adds a unit test as well.

### CODE REVIEW

**Comment by mansnils** [comment]
@felix-johnny can you review this?

**Comment by JonatanAntoni** [comment]
Can one of the admins verify this patch?

File: CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8.c [hunk 1]
```diff
@@ -19,10 +19,10 @@
 /* ----------------------------------------------------------------------
  * Project:      CMSIS NN Library
  * Title:        arm_depthwise_conv_s8.c
- * Description:	 s8 version of depthwise convolution.
+ * Description:  s8 version of depthwise convolution.
  *
- * $Date:        09. October 2020
- * $Revision:    V.2.0.1
+ * $Date:        11. May 2021
+ * $Revision:    V.2.0.2
```
Comment (felix-johnny) [inline CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8.c]: Just a minor comment..Since it is a feature addition, lets have it as 2.5.0.

Comment (mansnils) [reply CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8.c]: @felix-johnny it is already set to 2.5.0 in the second commit.

**Pull request merged**